opcodes/riscv-opc.c

```diff
@@ -82,9 +82,128 @@ const char * const riscv_fpr_names_abi[NFPR] =
 #define MASK_RL (OP_MASK_RL << OP_SH_RL)
 #define MASK_AQRL (MASK_AQ | MASK_RL)
 #define MASK_SHAMT (OP_MASK_SHAMT << OP_SH_SHAMT)
-#define MATCH_SHAMT_REV8_32 (0b11000 << OP_SH_SHAMT)
-#define MATCH_SHAMT_REV8_64 (0b111000 << OP_SH_SHAMT)
-#define MATCH_SHAMT_ORC_B (0b00111 << OP_SH_SHAMT)
+#define MATCH_SHAMT_REV_32     (0b11111 << OP_SH_SHAMT)
```

Comment:
I would suggest encode the (0b11111 << OP_SH_SHAMT) into riscv_opcodes directly, since these defines looks redundant.

Reply:
I am not sure how to do this, it is mean compute the result "MATCH_GORCI | MATCH_SHAMT_REV_32" and send it into riscv_opcodes or compute the result (0b11111 << OP_SH_SHAMT) and send it "MATCH_GORCI | res" ?